10x/README.md

```diff
@@ -35,7 +35,7 @@ cd skylab/10x/wdls/
   (constructed with `skylab/10x/generate_reference_bundle.wdl`)
 
 ## Runtime Requirements
-- Memory: 20 GB
+- Memory: 12 GB for testing data with a single-chromosome testing index. datasets with more reads or more sequencing lanes will require more memory. Note that this pipeline automatically scales horizontally to align reads with numerous processors, and each processor must load its own STAR index. This can very quickly results in out-of-memory errors. More than 4 lanes should not be tested on a laptop, even with the testing index and more than 2 are not recommended. 
```

Comment:
I would say chromosome 21 rather than single-chromosome. And very minor: datasets --> Datasets, "results in" --> "result in"